src/settings.py

```diff
@@ -77,11 +77,14 @@ class Settings(BaseSettings):
     COMMIT_DATE: str = ""
     TAGS: list[str] = []
 
+    # URLs проекта Procharity
+    PROCHARITY_URL: str
```

Comment:
Не вполне очевидно, как задать нужные значения? Передаём через `.env`? Тогда надо добавить информацию в `env.example`

Reply:
Добавь дефолтное значение. Также стоит добавить [валидатор](https://docs.pydantic.dev/2.1/api/networks/#pydantic.networks.AnyHttpUrl), чтобы в случае если предоставили не валидный url, приложение валилось сразу. 
```suggestion
    PROCHARITY_URL: AnyHttpUrl = "https://procharity.ru"
```